### DISCUSSION TITLE
where is this wallpaper?

### DISCUSSION THREAD

**Discussion by ItsTerm1n4l** [prasanthrangan/hyprdots#1747] [Q&A]
I have not been able to find the wallpaper you used in this image (from the readme)

![](https://raw.githubusercontent.com/prasanthrangan/hyprdots/main/Source/assets/rofi_style_12.png)

where can I find it?
Thank you!

**Comment by ItsTerm1n4l** [comment]
oops, I didn't see the discussions tab.

**Reply by Sofitt** [reply]
> oops, I didn't see the discussions tab.

So did you find this wallpaper?

**Reply by ItsTerm1n4l** [reply]
nope.

**Reply by kRHYME7** [reply]
check https://github.com/hyde-project/hyde-gallery 

It lists all the themes available, check each, and there should be wallpapers in there.

**Reply by Sofitt** [reply]
> check https://github.com/hyde-project/hyde-gallery
> 
> It lists all the themes available, check each, and there should be wallpapers in there.

If only it were that easy :)
Unfortunately this wallpaper is not there
Also prasanthrangan said he doesn't remember the origin because all these wallpapers were taken from a reddit threads

**Reply by kRHYME7** [reply]
I guess it is on his custom wallpaper directory.

**Reply by live4thamuzik** [reply]
If memory serves me well.... check /boot/grub/themes I believe that wallpaper is used for the Candy theme. I remember finding it before.
